airbyte-integrations/connectors/source-zendesk-sunshine/source_zendesk_sunshine/streams.py

```diff
@@ -77,6 +77,8 @@ def get_updated_state(self, current_stream_state: MutableMapping[str, Any], late
 
 
 class ObjectTypes(SunshineStream):
+    primary_key = "key"
```

Comment:
Is it possible to have the `primary_key` property mainly in `SunshineStream` and re-assign only if any other stream has something else but `key` ?

Reply:
There is set the `primary_key = "id"` in the `SunshineStream`. And if any other stream has another `primary_key`, it is overwritten.